safedata_validator/summary.py

```diff
@@ -326,7 +340,13 @@ def load(
         self._load_funders()
         self._load_permits()
         self._load_external_files()
-        self._load_data_worksheets(sheetnames)
+        self._load_sequenced_taxa_sheets(sheetnames=sheetnames)
+        self._load_data_worksheets(
+            sheetnames=sheetnames,
+            sequenced_taxa_sheets={
+                metadata["sheet_name"] for metadata in self.sequenced_taxa_metadata
+            },
+        )
```

Comment:
I sort of feel that at this point we should move `sheetnames` into an attribute and then handle all of this checking by accessing attributes from within the methods rather than passing stuff by argument. But that's really just the aesthetics of these private methods - the actual functionality is right, I think 😄

Reply:
I do think that's worth implementing actually. Just from the perspective of having to return to this bit of code in a year+, all the methods having a consistent pattern makes it easier to grasp what's going on

Reply:
Yup - then load immediately stores the full set of sheet names and the private methods can use it to check nothing is being left out or used twice etc etc.